test/hss_test.c

```diff
@@ -180,8 +180,11 @@ static int hss_verify_bad_sig_test(void)
      * Corrupt every 3rd byte to run less tests. The smallest element of an XDR
      * encoding is 4 bytes, so this will corrupt every element.
      */
+#if defined(OSSL_SANITIZE_MEMORY)
```

Comment:
Add a comment explaining that memory sanitisation makes this too slow so a larger step has to be used?

Reply:
I could probably take it out now that it no longer hangs :)

Reply:
If the amount of time stays reasonable then sure.